frontend/src/tests/components/mine/Incidents/__snapshots__/MineIncidentTable.spec.js.snap

```diff
@@ -374,6 +376,7 @@ ShallowWrapper {
                 "key": undefined,
                 "mine_incident_report_no": "2019-70",
                 "openMineIncidentModal": [MockFunction],
+                "openViewMineIncidentModal": undefined,
```

Comment:
Looks like mocks or props need to be updated

Reply:
100%, Haven't touched the tests yet.